src/server/handlers/manager/subnet.rs

```diff
@@ -4,24 +4,26 @@
 
 use crate::config::{ReloadableConfig, Subnet};
 use crate::jsonrpc::{JsonRpcClient, JsonRpcClientImpl};
-use crate::manager::LotusSubnetManager;
+use crate::manager::{LotusSubnetManager, SubnetManager};
 use ipc_identity::Wallet;
 use ipc_sdk::subnet_id::SubnetID;
 use std::sync::{Arc, RwLock};
 
 /// The subnet manager connection that holds the subnet config and the manager instance.
-pub struct Connection<T: JsonRpcClient> {
+pub struct Connection {
     subnet: Subnet,
-    manager: LotusSubnetManager<T>,
+    manager: Box<dyn SubnetManager>,
```

Comment:
Why not pass SubnetManager as generic parameter but use Box + dyn?

Reply:
Oh, I forgot to push the version where I was using generics. That works, but good catch, thanks!